examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -510,14 +510,14 @@ def act(self, obs, env):
                 "agent_0_oracle_nav_action"
             ]._path_to_point(human_pos)
             # Sanitize the policy input even thought we did not call that here
-            self._init_policy_input()
+            self._policy_info = self._init_policy_input()
```

Comment:
Why do we sanitize (self._policy_info = self._init_policy_input()) the policy at every act call?